tests/functional/test_contract_instance.py

```diff
@@ -54,6 +54,11 @@ def test_contract_calls(owner, contract_instance):
     assert contract_instance.myNumber() == 2
 
 
+def test_contract_fixture(contract_instance):
```

Comment:
no this is not testing the fixture, you need to add an integration test.
add a test to this file instead: https://github.com/ApeWorX/ape/blob/main/tests/integration/cli/projects/test/tests/test_fixtures.py